certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -243,7 +243,7 @@ def _deploy_cert(self, vhost, cert_path, key_path, chain_path, fullchain_path):
 
         self.parser.update_or_add_server_directives(vhost,
                                           cert_directives)
-        logger.info("Deploying Certificate to VirtualHost %s", vhost.filep)
+        logger.info("Deploying Certificate to %s", vhost.filep)
```

Comment:
@ohemorange as I mentioned in chat, the site that causes the looped printing is at:

https://github.com/certbot/certbot/blob/e378931eda6ce6bcd9059959c4b3b790f41a203d/certbot/certbot/_internal/client.py#L514-L516

I don't think we can change the installer interface. We could demote to debug, unless you feel that users find this useful for the success cases?